amarillo/models/Carpool.py

```diff
@@ -195,6 +280,17 @@ class Carpool(BaseModel):
         max_length=20,
         pattern='^[a-zA-Z0-9]+$',
         examples=["mfdz"])
+    
+    driver: Optional[Driver] = Field(
+        None, 
+        description="Driver data", 
+        examples=["""
+                {
+                    "driver_id": "123", 
+                    "profile_picture": "https://mfdz.de/driver/789/picture",
+                    "rating": 5
+                }
+                """])
```

Comment:
This is also an addition to the GTFS Standard. why isn't this part of additional_ridesharing_info?

Reply:
It's part of driver.txt